controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -288,6 +289,7 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 					return []reconcile.Request{}
 				},
 			),
+			builder.WithPredicates(predicate.GenerationChangedPredicate{}),
```

Comment:
I don't believe sc has generation field.

Reply:
removed it.